app/store/message.go

```diff
@@ -138,3 +131,31 @@ func (s *Session) GetMessages(i int) *Message {
 func (m *Message) GetName() string {
 	return TelToName(m.Source)
 }
+
+// FindQuotedMessage searches the equivalent message of DataMessage_Quote in our
+// DB and returns the local message id
+func FindQuotedMessage(quote *signalservice.DataMessage_Quote) (error, int64) {
+	var quotedMessages = []Message{}
+	err := DS.Dbx.Select(&quotedMessages, "SELECT * FROM messages WHERE sentat = ?", quote.GetId())
+	if err != nil {
+		return err, -1
+	}
+	if len(quotedMessages) == 0 {
+		return err, -1
```

Comment:
err here can be nil, maybe you should write a real message?

Reply:
good catch!